eisbuk-admin/src/store/selectors/customers.ts

```diff
@@ -22,3 +23,46 @@ export const getCustomersList = (state: LocalStore): Customer[] => {
   const customersInStore = getCustomersRecord(state);
   return Object.values(customersInStore);
 };
+
+/**
+ * Get a list of all the customers whose birthdays are today
+ * (not an actual firestore query but rather synced local store entry)
+ * @param state Local Redux Store
+ * @returns list of customers grouped by birthday
+ */
+export const getCustomersWithBirthday =
+  (date: DateTime) =>
+  (state: LocalStore): CustomerBirthday[] => {
+    const customersInStore = getCustomersRecord(state);
+
+    const customers: CustomerBirthday[] = [];
+    Object.values(customersInStore).forEach((customer) => {
+      const index = customers.findIndex(
+        (customerBirthday) => customerBirthday.birthday === customer.birthday
+      );
+      index !== -1
+        ? customers[index].customers.push(customer)
+        : customers.push({
+            birthday: customer.birthday,
+            customers: [customer],
+          });
+    });
+    const sortedCustomers = customers.sort((a, b) =>
+      DateTime.fromISO(a.birthday)
+        .set({ year: 2000 })
+        .toString()
+        .localeCompare(
+          DateTime.fromISO(b.birthday).set({ year: 2000 }).toString()
+        )
+    );
+    const index = sortedCustomers.findIndex(
+      (c) =>
+        date.startOf("day").set({ year: 2000 }) <=
+        DateTime.fromISO(c.birthday).startOf("day").set({ year: 2000 })
+    );
```

Comment:
This could just be string comparison (substr(5)) which will give you `mm-dd` from `yyyy-mm-dd` and then two of those are comparable (as dates within a year)